### ISSUE TITLE
nvidia gpu device acess in containers

### ISSUE THREAD

**Issue by v-snap** [NVIDIA/libnvidia-container#287]
Hi I am facing an issue in setting up the gpu inside container.

**OS: ubuntu 22.04
podman versions : 3.4.4
nvidia-ctk version : 1.17**

Followed all the instruction as per the below link but still no success : 
`https://github.com/NVIDIA/nvidia-container-runtime/issues/85#issuecomment-1772741424`
 
- nvidia-ctk cdi generate --output=/etc/cdi/nvidia.yaml

- podman run --rm --device nvidia.com/gpu=all --security-opt=label=disable ubuntu nvidia-smi -L

```Error: stat nvidia.com/gpu=all: no such file or directory```

**Comment by ericthomasca** [comment]
Did you try running `sudo nvidia-ctk cdi generate --output=/etc/cdi/nvidia.yaml`?

**Comment by viprob-ai** [comment]
Yes after trying many things with no success finally I upgraded the Podman version to 5.0 and everything works.
